### PR TITLE
Add bdn subnet without gateway for testing

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

**Comment by brtkwr** [comment]
When we eventually deploy swarm-sip with fip disabled, we are going to need a bdn network without gateway. For that reason, isn't this worth merging?

File: etc/p3-config/p3-config.yml [hunk 1]
```diff
@@ -469,14 +477,14 @@ p3_flavor_storage_B:
...
-  - "{{ p3_image_mlnx_ofed }}"
-  - "{{ p3_image_anaconda }}"
+#  - "{{ p3_image_centos }}"
+#  - "{{ p3_image_ubuntu }}"
+#  - "{{ p3_image_fedora }}"
+#  - "{{ p3_image_openhpc }}"
+#  - "{{ p3_image_centos_cuda }}"
+#  - "{{ p3_image_ubuntu_cuda }}"
+#  - "{{ p3_image_mlnx_ofed }}"
+#  - "{{ p3_image_anaconda }}"
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml]: Uncomment me

**Comment by markgoddard** [comment]
Why does the lack of a floating IP have an effect here?

**Comment by brtkwr** [comment]
It doesn't, but I suppose this network is necessary as part of swarm-sip p3-appliances anyway, not just a test case, so makes sense to hard code it in.

File: etc/p3-config/p3-config.yml (hunk 1 above)
Comment (brtkwr) [reply etc/p3-config/p3-config.yml]: Done

**Review by oneswig — APPROVED** [review]
I don't recall how these two tasks got tangled but in isolation they both appear fine...

File: etc/p3-config/p3-config.yml [hunk 2]
```diff
@@ -288,6 +297,7 @@ p3_networks:
   - "{{ p3_network_ilab }}"
   - "{{ p3_network_internal }}"
   - "{{ p3_network_bdn }}"
+  - "{{ p3_network_nogwateway }}"
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml:300]: there is a typo here - nogwateway

**Pull request merged**